src/pytest_plugins/consume/simulator_common.py

```diff
@@ -36,3 +36,18 @@ def fixture(fixture_source: JsonSource, test_case: TestCase) -> Fixture:
         fixtures = BlockchainFixtures.from_file(Path(fixture_source) / test_case.json_path)
         fixture = fixtures[test_case.id]
     return fixture
+
+
+@pytest.fixture(scope="function")
+def fixture_description(fixture: Fixture, test_case: TestCase) -> str:
+    """
+    Return the description of the current test case.
+    """
+    description = f"Test id: {test_case.id}"
+    if "url" in fixture.info:
+        description += f"\n\nTest source: {fixture.info['url']}"
+    if "description" not in fixture.info:
+        description += "\n\nNo description field provided in the fixture's 'info' section."
+    else:
+        description += f"\n\n{fixture.info['description']}"
```

Comment:
Personally I prefer having only the test ID and URL. It would be amazing to have the test descriptions but I dont find the tradeoff of adding it to the fixture to be worth it

I'm open to it if others like it though

Reply:
(will wait and see what the generated fixtures look like, maybe it will change my mind)

Reply:
https://github.com/ethereum/execution-spec-tests/pull/579#issuecomment-2139749086